server/src/main/java/org/opensearch/common/remote/RemoteWriteableEntityBlobStore.java

```diff
@@ -89,6 +94,17 @@ public void readAsync(final U entity, final ActionListener<T> listener) {
         });
     }
 
+    public void readAsyncWithDelay(final long delayMillis, final U entity, final ActionListener<T> listener) {
+        Runnable runnable = () -> {
+            try {
+                listener.onResponse(read(entity));
+            } catch (Exception e) {
+                listener.onFailure(e);
+            }
+        };
+        threadPool.scheduleUnlessShuttingDown(TimeValue.timeValueMillis(delayMillis), executor, runnable);
+    }
+
```

Comment:
Shouldn't this jitter handling be done on the remote store side, by making it configurable at the repository level

Reply:
Removed this piece of code.